ui/src/shared/components/cells/Cells.tsx

```diff
@@ -65,16 +80,24 @@ class Cells extends Component<Props & WithRouterProps> {
   }
 
   private get cells(): Layout[] {
+    const {state} = this.props
     return this.props.cells
       .filter(c => c.status === RemoteDataState.Done)
-      .map(c => ({
-        ...c,
-        x: c.x,
-        y: c.y,
-        h: c.h,
-        w: c.w,
-        i: c.id,
-      }))
+      .map(c => {
+        const view = getByID<View>(state, ResourceType.Views, c.id)
+        let cell = {
+          ...c,
+          x: c.x,
+          y: c.y,
+          h: c.h,
+          w: c.w,
+          i: c.id,
+        }
+        if (view.properties.type === 'gauge') {
+          cell = {...cell, minW: 3, minH: 3}
```

Comment:
instead of using the spread operator here, we can simply say:

```
cell.minW = 3
cell.minH = 3
```

This will also be true if the variable is a constant

Reply:
That did not work... it seemed quite angry. Is it supposed to be cell= {cell.minH etc...}

Reply:
by reassigning the minH and minW, we be overwriting the actual object, so in the end the cell would only have the properties that we've reassigned. What's the error saying? It could be a TypeScript error that's being protective about cell being too loose a type, in which case it would be best for us to create a type for the cell